bolt/lib/Passes/IdenticalCodeFolding.cpp

```diff
@@ -340,6 +354,113 @@ typedef std::unordered_map<BinaryFunction *, std::vector<BinaryFunction *>,
 
 namespace llvm {
 namespace bolt {
+/// Scans symbol table and creates a bit vector of memory addresses of vtables.
```

Comment:
Refactor the code that handles vtable addresses. E.g. add a function `isInVTable(uint64_t)`. You can make the bit vector a member of the ICF pass class.